CHANGES.md

```diff
@@ -1,3 +1,20 @@
+## edge-19.5.3
+
+* CLI
+  * **New** Added a `linkerd edges` command that shows TLS status
```

Comment:
I think we're trying to differentiate between TLS and identity in general, so I think it would be better to say "Added a `linkerd edges` command that shows connections between src and dst resources and the identities of their Linkerd proxies. This will help users verify that Linkerd proxies have a known identity."

Reply:
Update, recommendation from @klingerf: "Added a `linkerd edges` command that shows the source and destination name and identity for proxied connections, to assist in debugging" I think that's more clear.

Reply:
thanks!